test_mpop_service/TestMpopService.h

```diff
@@ -40,9 +40,12 @@ private slots:
     // called after the last test function was executed
     void cleanupTestCase();
 
+    //called testentries function to delete all the test entries
+    void removeDatabaseTestEntries();
+
 private:
     QSharedPointer<Facade> facade;
     bool is_mysql_supported;
 
-    void removeDatabaseTestEntries();
+    //void removeDatabaseTestEntries();
```

Comment:
We should not comment out the declaration of removeDatabaseTestEntries(), since we need it.